plugins/sqlfluff-templater-dbt/sqlfluff_templater_dbt/templater.py

```diff
@@ -439,3 +441,9 @@ def _unsafe_process(self, fname, in_str=None, config=None):
             # No violations returned in this way.
             [],
         )
+
+    @classmethod
+    def _slice_template(cls, in_str: str) -> List[RawFileSlice]:
+        # DbtTemplater uses the original heuristic-based template slicer.
+        # TODO: Can it be updated to use TemplateTracer?
+        return slice_template(in_str, cls._get_jinja_env())
```

Comment:
I can do that. I have some worries that dbt may not expose the Jinja environment, etc. necessary to do this. I'll do some preliminary exploration (not coding) before merging this PR.

Reply:
About the separate module, do you mean a separate Python package? Note that it is a separate module now, in the sense of a separate `.py` file.

I'm hesitant to create a separate package because I worry I'd end up being one person supporting a package with lots of users. I was in that situation several years ago, and it was difficult for me. What do you think? (https://github.com/AirSage/Petrel)

Reply:
I created an issue for the dbt work: #1783